doc/release-notes/9434-app-container.md

```diff
@@ -0,0 +1,5 @@
+Developers can experiment with running Dataverse in Docker: https://preview.guides.gdcc.io/en/develop/container/app-image.html
+
+This is an image developers build locally. It is not meant for production use!
```

Comment:
Why, is there a rationale? If there is a docker-image available people will start to use it in various contexts

Reply:
Yes. IQSS doesn't use containers at this point for development, let alone production. This is Open Source Software, and very often people using OSS have a tendency to expect quick and free help for all sorts of stuff. So we need to emphasize what this is meant to be and draw some lines in the sand. Basically this is saying "You're on your own, but you may ask humble and politely for help while exploring".

Reply:
Yes, exactly. Please be patient with folks at IQSS. Some of us have only installed Docker last week.

I think the wording is pretty good. It sets expectations properly.